features

```diff
@@ -0,0 +1,109 @@
+#!/usr/bin/env python
+# coding: utf-8
+
+# In[8]:
+
+
+
+import pickle
+import pandas as pd
+
+
+#loads file
+datafolder = "/Users/chaiqifei/Documents/Group Project/"
+filename = "file.pkl"
+with open(datafolder+filename, "rb") as fle:
+  df = pickle.load(fle)
+
+
+# In[47]:
+
+
+from IPython.display import display
+pd.options.display.max_columns = None
+df.head()
+
+
+# In[24]:
+
+
+#keep only those companies whose T0 status is active
+df = df[df['CompanyStatus_T0']=='Active']
+import numpy as np
+#isolate target outcome - company credit status at time T1
+target = np.array(df['class_1'])
+
+
+# In[73]:
+
+
+features = df[['CompanyNumber', 'RegAddressPostCode',
+        'CountryOfOrigin','MortgagesNumMortCharges',
+       'MortgagesNumMortOutstanding', 'MortgagesNumMortPartSatisfied', 'LimitedPartnershipsNumGenPartners',
+       'LimitedPartnershipsNumLimPartners',  'SIC1',
+       'eCompanyCategory', 'eCompanyStatus',
+       'eAccountsAccountCategory', 'dIncorporationDate',
+       'dAccountsNextDueDate',
+       'dReturnsNextDueDate','dConfStmtNextDueDate','namechanged', 'namechanged2']].copy()
```

Comment:
features = df[['CompanyNumber', 'oa11',
        'CountryOfOrigin','MortgagesNumMortCharges',
       'MortgagesNumMortOutstanding', 'MortgagesNumMortPartSatisfied', 'LimitedPartnershipsNumGenPartners',
       'LimitedPartnershipsNumLimPartners',  'SIC1',
       'eCompanyCategory', 'eCompanyStatus',
       'eAccountsAccountCategory', 'dIncorporationDate',
       'dAccountsNextDueDate',
       'dReturnsNextDueDate','dConfStmtNextDueDate','namechanged', 'namechanged2', 'class_1']].copy()

Replaced postcode with area code and added class_1 to be extracted later